src/erlcloud_kinesis.erl

```diff
@@ -325,20 +325,31 @@ get_records(ShardIterator, Limit)
     get_records(ShardIterator, Limit, default_config()).
 
 -spec get_records/3 :: (string(), 1..10000, aws_config()) -> proplist().
-get_records(ShardIterator, Limit, Config) when is_record(Config, aws_config),
-                                               is_integer(Limit),
-                                               Limit > 0,
-                                               Limit =< 10000 ->
-    Json = [{<<"ShardIterator">>, ShardIterator}, {<<"Limit">>, Limit}],
-    get_normalized_records(Config, Json).
-
-%% Normalize records from Kinesis
-
-get_normalized_records(Config, Json) when is_record(Config, aws_config) ->
-  case erlcloud_kinesis_impl:request(Config, "Kinesis_20131202.GetRecords", Json) of
-    {ok, Response} -> {ok, normalize_response(Response)};
-    {error, Msg} -> {error, Msg}
-  end.
+get_records(ShardIterator, Limit, Config) ->
+    get_records(ShardIterator, Limit, false, Config).
+
+-spec get_records/4 :: (binary(), 1..10000, boolean(), aws_config()) ->
+    {ok, [proplists:proplist()] | binary()} | {error, any()}.
+get_records(ShardIterator, Limit, IsRawResp, Config)
```

Comment:
I wonder if `IsRawResp` should actually be a proplist of erlcloud options?
I think this would be more consistent with how other options are passed within erlcloud, and would look nicer as a consumer
`erlcloud_kinesis:get_records(ShardIt, Limit, [{no_decode, true}], AwsConfig)`
vrs the more opaque
 `erlcloud_kinesis:get_records(ShardIt, Limit, true, AwsConfig)`